src/__tests__/homepage.test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import {cleanup, render, screen} from '@testing-library/react';
+import HomePage from "../pages/Home/Home.page";
+import youtubeVideos from "../mocks/youtube-videos";
+
+describe('Home page Tests', () => {
+  afterEach(cleanup);
+
+  describe('Ensure elements get rendered', () => {
+    it('Title', () => {
+      render(<HomePage />);
+      expect(screen.getByText(/YouTube video search app/, {selector: 'h1'})).toBeInTheDocument();
```

Comment:
Prefer the `getByRole` query in this case with `heading` role.

Reply:
Thanks, I will change it.